### DISCUSSION TITLE
Welcome to wiz_light Discussions!

### DISCUSSION THREAD

**Discussion by sbidy** [sbidy/wiz_light#61] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by caesar1987sk** [comment]
I have tested latest version and it's working. I have only one question. Wasn't there combo for select scene on light card? I never selected any scene, but maybe some people do that.

**Reply by sbidy** [reply]
I'm not sure, but the default light card doesn't support that. So maybe I'll create a complete new type of entity.
Currently the wiz_light is derived from the light default entity type.

**Reply by XRyu** [reply]
Its not in the default light card. If you want to use the scenes i recommend the "light-entity-card", installable via HACS.
with that one, you have many options to represent the light and show settings. :)
i use it for different kinds of lights:
![image](https://user-images.githubusercontent.com/131995/103141346-66c33b00-46f3-11eb-9aaa-eeacac91456d.png)
![image](https://user-images.githubusercontent.com/131995/103141352-7e9abf00-46f3-11eb-9566-ac21ef075223.png)

**Reply by caesar1987sk** [reply]
Thank you. I don't need it, I was just not sure if that wasn't there. I have created buttons for custom values 🙂 and that's what I'm using, usually I don't even open light card.

**Comment by OptimusGREEN** [comment]
Hi,

I've never actually used any of these bulbs before and I'm considering buying some. I have a few questions before hand though.

1: are these cloud based bulbs?

2: does this component work locally or via the cloud

3: is the wifi in them any good or is it a bit flaky? I have tcp wifi bulbs that are a bit flaky (tuya based) now on tasmota though.

I'm sure i probably have more questions that i can't think of just now lol.

Thanks in advance.

**Reply by caesar1987sk** [reply]
My bulbs are working well, they have better WiFi connection that baby camera😂. Using this integration they are working fast and over the lan.

**Comment by rodriguecarneiro33** [comment]
Hey Folks!
First, thank you so much for the contributors of this integration, it's really awesome! 

I did install everything a month ago and everything was working perfectly (I only have bulb from WiZ). 

I changed my router and Modem yesterday and updated to the last version of HA and now the integration isn't working anymore...

I get this message: 
![image](https://user-images.githubusercontent.com/3332303/103934761-239d9a80-50eb-11eb-8810-fe8be7739eaf.png)

I deleted the custom components, re-installed it and changed the IP on the configurations file as my IP changed from 10.0.0.XX to 192.168.1.XX

I don't know where the error could come from now, any ideas? Here is my configuration file: 
![image](https://user-images.githubusercontent.com/3332303/103934934-72e3cb00-50eb-11eb-868d-cede3f0f5ab6.png)

**Reply by caesar1987sk** [reply]
When you have removed the custom component, added it back, then you need to remove (comment out)also configuration for bulbs, restart hassio and then you can add back configuration of bulbs, restart hassio again and then it should work.

**Reply by caesar1987sk** [reply]
So steps are
1. When custom component is not in folder copy it there. 
2. Remove bulbs config
3. Restart hassio
4. Add bulbs config
5. Restart hassio

Primary problem on beginning was changed ip of of all bulbs on new router.

**Reply by rodriguecarneiro33** [reply]
Thank you so much @caesar1987sk , it did work perfectly!
